UnitTests/GitCommandsTests/Remote/GitRemoteManagerTests.cs

```diff
@@ -249,5 +249,105 @@ public void SetRemoteState_should_call_ToggleRemoteState(string remoteName, bool
             _configFile.Received(1).AddConfigSection(sections[remoteDisabled ? 1 : 0]);
             _configFile.Received(1).Save();
         }
+
+        [Test]
+        public void ConfigureRemotes_Should_not_update_localHead_if_remoteHead_is_local()
+        {
+            var refs = new List<IGitRef>
+            {
+                CreateSubstitureRef("f6323b8e80f96dff017dd14bdb28a576556adab4", "refs/heads/local", ""),
+
+            };
+            _module.GetRefs().ReturnsForAnyArgs(refs);
+
+            _controller.ConfigureRemotes("origin");
+
+            var mergeWith = "";
+            Assert.AreEqual(refs[0].MergeWith, mergeWith);
```

Comment:
Arguments are out of order. Should be `AreEqual(expected, actual)`.

This is true in several of the tests you've added.

When they're out of order and tests fail then the error message is confusing (_expected "foo" but was ""_, rather than _expected "" but was "foo"_).